packages/shared/src/authz/domain/workspaces/operations.ts

```diff
@@ -17,7 +16,6 @@ export type GetWorkspace = (args: {
   Result<
     Workspace,
     | InstanceType<typeof WorkspaceNotFoundError>
-    | InstanceType<typeof WorkspaceSsoSessionNoAccessError>
```

Comment:
that error has to remain there, because it can be thrown when the FE invokes the getWorkspace loader